app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecordInfo.kt

```diff
@@ -0,0 +1,16 @@
+package org.simple.clinic.teleconsultlog.teleconsultrecord
+
+import org.simple.clinic.medicalhistory.Answer
```

Comment:
We should not use the Medical history answer here and should create a dedicated type for this. They are two different concepts and should be represented differently, even if they look the same.